### ISSUE TITLE
GtDictionnaryString.py and modded minecraft

### ISSUE THREAD

**Issue by Ex0danify** [PoroCoco/myaenetwork-Server#3]
Hey there,

Do I need to re-generate said GtDictionnaryString.py for my own Minecraft server instance?
I'm running about 300+ mods and have tons of different Items not listed in the supplied GtDictionnaryString.py.
Is there any easy way to auto-generate this?

Thanks for your awesome work!
Cheers

**Comment by PoroCoco** [comment]
Hello !

Thanks for the message.

The GtDictionnaryString.py is only useful if you have the Gregtech mod inside your modpack as it's used to translate the cryptic Gregtech names into readable ones. If you have Gregtech in your modpack I can give you the script to generate it from the Gregtech.lang file.
If you do not have Gregtech into your modpack then you don't need it and should change the line 19 of server.py from 
`translateGTName = True`
to 
`translateGTName = False`

I'm currently working on remaking the code for the server and the ingame computers as it hurts to look at the code I wrote here, it has really bad performances and there is no documentation. I also closed the official website as there was no people using it anymore.

If you encounter any other problems feel free to ask me !
